### ISSUE TITLE
Double quotes added for fields with single quote

### ISSUE THREAD

**Issue by jheeffer** [paulfitz/daff#105]
## Test case:

input.csv:
```
the IMF's Balance of Payments Manual (BPM6),foo,"bar",baz',"boo""z"
```

command:
```
daff merge --output output.csv input.csv input.csv input.csv
```

output.csv:
```
"the IMF's Balance of Payments Manual (BPM6)",foo,bar,"baz'","boo""z"
```

Expected output:
```
the IMF's Balance of Payments Manual (BPM6),foo,bar,baz',"boo""z"
```

## Reasoning
Single quotes don't need escaping nor double quoting of the containing field according to the RFC. For consistency it would be nice to have `daff` follow the RFC. It would mean we can have clean daff git merges of CSVs created by other RFC-compliant code.

**Comment by jheeffer** [comment]
Seems it would be as simple as removing `'\''` from https://github.com/paulfitz/daff/blob/master/coopy/Csv.hx#L93 ?
Question is of course if this change would break code that relies on this for some (strange) reason.

**Comment by jheeffer** [comment]
Thanks! Looks good! 

For completeness: the rfc says that you may quote any field, so also a field with single quote. You can define different quoting strategies. With this change daff is following a "minimal quoting" strategy: only quote when needed. Before it was more of a "custom" strategy I guess ;). Other examples of strategies are quote all or quote non numeric. But just following one strategy is probably enough for daff (it is for our use case).

Also see python manual (search QUOTE_ALL): https://docs.python.org/3/library/csv.html#csv-contents

**Comment by paulfitz** [comment]
Merged in 1.3.29, please re-open if there's a goof.
